ocs_ci/helpers/dr_helpers.py

```diff
@@ -1709,11 +1722,25 @@ def replace_cluster(workload, primary_cluster_name, secondary_cluster_name):
     if not sample.wait_for_func_status(result=True):
         raise Exception("Namespace openshift-operators is not created")
 
-    # add label to openshift-opeartors namespace
+    cbp_obj = ocp.OCP(
+        kind=constants.CEPHBLOCKPOOL,
+        resource_name=constants.DEFAULT_CEPHBLOCKPOOL,
+        namespace=config.ENV_DATA["cluster_namespace"],
+    )
+    mirroring_status = cbp_obj.get().get("status").get("mirroringStatus")
+    if mirroring_status:
+        raise Exception("Mirroring is not disabled completely")
+        logger.info(f"Mirroring status: {mirroring_status}")
```

Comment:
is there something to sample from CephFS/volsync side?

Reply:
Currently, we support it for RBD based workload. This has to be handled for cephFS when we support it in future.